java-squid/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -63,12 +61,14 @@
 
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
-import java.io.PrintStream;
+
 import java.util.ArrayList;
 import java.util.Deque;
+import java.util.HashSet;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class CFG {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this class is 205 which is greater than 200 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AClassCyclomaticComplexity)